lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -256,6 +261,8 @@ def build_entity(self):
             name=self._module.params['name'],
             description=self._module.params['description'],
             comment=self._module.params['comment'],
+            import_=True if (self._module.params['state'] == 'import' and storage_type in ['iscsi', 'fcp']) else None,
```

Comment:
because I was trying to reflect the same behavior as being used also in the REST.
We use an "_import" boolean flag for block storage domains to tell whether it is a new storage domain or an imported one, because with FCP for example we don't mention any targets at all *that is why we added a new flag "_import")
I can use this flag as well for other storage domains if you think that it will be necessary